db/schema.rb

```diff
@@ -234,8 +234,8 @@
     t.datetime "invitation_sent_at"
     t.datetime "invitation_accepted_at"
     t.integer  "invitation_limit"
-    t.integer  "invited_by_id"
     t.string   "invited_by_type"
+    t.integer  "invited_by_id"
```

Comment:
Please remove this file from the PR.

Reply:
I can't comment on the package-lock.json file for some reason, but please also remove that file too.

Reply:
noted.